integration-cli/docker_cli_run_test.go

```diff
@@ -2718,6 +2718,20 @@ func TestRunAllowPortRangeThroughExpose(t *testing.T) {
 	logDone("run - allow port range through --expose flag")
 }
 
+// test docker run expose a invalid port
+func TestRunExposePort(t *testing.T) {
+	runCmd := exec.Command(dockerBinary, "run", "--expose", "80000", "busybox")
+	_, _, err := runCommandWithOutput(runCmd)
+	//expose a invalid port should with a error out
+	if err == nil {
```

Comment:
So with the above comment, this would become:

```go
if err == nil || !strings.Contains("Invalid range format for") {
    t.Fatalf("run --expose a invalid port should with error out\n%s", out)
}
```